package.json

```diff
@@ -105,5 +105,9 @@
       ],
       "@semantic-release/github"
     ]
-  }
+  },
+  "sideEffects": [
+    "./style/default.css",
+    "./style/default.min.css"
+  ]
```

Comment:
@andreafalzetti Thanks for your comment. I heard about that but I didn't use yet, but my friend used and said that its great! Sorry, but I was in Friday off.

Before merge - it should be moved to the https://github.com/asyncapi/asyncapi-react/blob/master/library/package.json place (package.json of component) - here is a package.json where we have shared dependencies across project (we use the lerna for multirepo).

Reply:
Makes sense, good catch! Moved. Check in 60117f08d0790c0d77f6422db53789633ee0d2cb